packages/styling/src/styles/DefaultTypography.ts

```diff
@@ -0,0 +1,73 @@
+import { DefaultFontStyles } from './DefaultFontStyles';
+import { FontWeights } from './fonts';
+import { IFontFamilies, IFontSizes, IFontWeights, IFontTypes, ITypography } from '../interfaces/ITypography';
+
+export const DefaultFontSizes: IFontSizes = {
+  tiny: '1rem',
```

Comment:
this we should probably make sure we all agree on names and default sizing (@mikewheaton @Jahnp)